company-tng.el

```diff
@@ -132,6 +140,11 @@ confirm the selection and finish the completion."
   (setq company-frontends '(company-tng-frontend
                             company-pseudo-tooltip-frontend
                             company-echo-metadata-frontend))
+  (setq company-clang-insert-arguments nil
+        company-semantic-insert-arguments nil
+        company-rtags-insert-arguments nil
+        lsp-enable-snippet nil)
```

Comment:
`lsp-enable-snippet` won't have any effect since the snippets are already enabled on server startup.

Reply:
Wouldn't it? This function is supposed to be called from the user's init script, so a restart should fix the problem, at least.

Reply:
Maybe you should also disable `company-lsp-enable-snippet`? This way even if LSP sends a snippet, `company-lsp` won't insert it.